tools/scripts/docs-website/create-docs-pr.js

```diff
@@ -141,36 +148,54 @@ function validateTag (version, force) {
 async function getReleaseNotes (version, releaseNotesFile) {
   console.log('Retrieving release notes from file: ', releaseNotesFile)
 
-  const data = await readReleaseNoteFile(process.cwd() + '/' + releaseNotesFile)
+  const data = await fs.readFile(process.cwd() + '/' + releaseNotesFile, 'utf8')
 
   const sections = data.split('\n## ')
   // Iterate over all past releases to find the version we want
   const versionChangeLog = sections.find((section) => section.startsWith(version))
   // e.g. v7.1.2 (2021-02-24)\n\n
   const body = versionChangeLog + SUPPORT_STATEMENT
   //   const [, releaseDate] = headingRegex.exec(versionChangeLog)
-
   // month and day should be in 2 digit format to allow for docs-site CI to run correctly
   const releaseDate = new Date().toLocaleDateString('sv')
 
   return { body, releaseDate }
 }
 
 /**
- * Reads the contents of NEWS.md
+ * Extracts the supported browser versions from the specified JSON file and creates a support string.
  *
- * @param {string} file path to NEWS.md
+ * @param {string} version The new version.
+ * @param {string} browsersFile The filename where the supported browsers JSON is stored.
  */
-async function readReleaseNoteFile (file) {
-  return new Promise((resolve, reject) => {
-    fs.readFile(file, 'utf8', (err, data) => {
-      if (err) {
-        return reject(err)
-      }
+async function getBrowserTargetStatement (version, browsersFile) {
+  console.log('Retrieving supported browser targets from file: ', browsersFile)
+
+  const browserData = await fs.readJson(process.cwd() + '/' + browsersFile)
+
+  const min = {}
+  const max = {}
+
+  for (let browser in browserData) {
+    const versions = browserData[browser]
+    min[browser] = Infinity
+    max[browser] = -Infinity
+    for (let browserVersion of versions) {
+      const versionNumber = Number(browserVersion.version)
+      min[browser] = min[browser] > versionNumber ? versionNumber : min[browser]
+      max[browser] = max[browser] < versionNumber ? versionNumber : max[browser]
+    }
+  }
 
-      return resolve(data)
-    })
-  })
+  const ANDROID_CHROME_VERSION = 100 // SauceLabs only offers one Android Chrome version
+
+  return (
+    'Consistent with our [browser support policy](https://docs.newrelic.com/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring/#browser-types), ' +
+    `version ${version} of the Browser agent was built for and tested against these browsers and version ranges: ` +
+    `Chrome ${min.chrome}-${max.chrome}, Edge ${min.edge}-${max.edge}, Safari ${min.safari}-${max.safari}, Firefox ${min.firefox}-${max.firefox}; ` +
+    `and for mobile devices, Android Chrome ${ANDROID_CHROME_VERSION} and iOS Safari ${min.ios}-${max.ios}. ` +
+    'Instrumentation and specific features may be compatible with other browsers or versions.'
```

Comment:
This part seems awkward, but i can't put my finger on it.  Is there a point to saying this if said "other browsers" are outside of our supported window?

Reply:
I've removed the last line as it's covered in the linked browser policy.